src/test/java/ancaMarian/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,110 @@
+package ancaMarian.tests.tema6;
+
+import ancaMarian.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import teofilursan.driver.BrowserManager;
+
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws InterruptedException {
+        navigateToDemoQAWindowPage();
+        clickOnNewTabButton();
+        closeBrowser();
+        navigateToDemoQAWindowPage();
+        clickOnNewWindowButton();
+        closeBrowser();
+        navigateToDemoQAWindowPage();
+        clickOnNewWindowMessageButton();
+        closeBrowser();
+        navigateToDemoQAWindowPage();
+        try {
+            clickOnNonExistingTable();
+        } catch (NoSuchElementException e){
+            FileUtils.takeScreenshot(driver, "ElementNotFound");
+            System.out.println("Nu s-a gasit elementul pe pagina. S-a salvat un screenshot.");
+        } finally {
+            closeBrowser();}
+    }
+
+    public static void navigateToDemoQAWindowPage() {
+        driver = BrowserManager.createChromeDriver();
```

Comment:
Replace BrowserManager.createChromeDriver() with BrowserManager.createDriver() - the method from your package. Thanks!